### PR TITLE
Effects menu entries

### PR DESCRIPTION
**Pull request by ksoze95**
Resolves: #3076

*(short description of the changes and the motivation to make the changes)*

- [x] I signed [CLA](https://www.audacityteam.org/cla/)
- [x] The title of the pull request describes an issue it addresses
- [x] If changes are extensive, then there is a sequence of easily reviewable commits
- [x] Each commit's message describes its purpose and effects
- [x] There are no behavior changes unnecessary for the stated purpose of the PR

Recommended:
- [x] Each commit compiles and runs on my machine without known undesirable changes of behavior

### CODE REVIEW

File: locale/af.po [hunk 1]
```diff
@@ -13739,7 +13739,7 @@ msgid "&Generate"
 msgstr "&Genereer"
 
 #: src/menus/PluginMenus.cpp
-msgid "Add / Remove Plug-ins..."
+msgid "Plugin Manager"
```
Comment (vsverchinsky) [inline locale/af.po:13742]: @Paul-Licameli is it necessary to update po files each time new msgid added or removed from the sources?

Comment (Paul-Licameli) [inline locale/af.po:13742]: The second commit in this sequence is quite unnecessary!  Remove it.

As part of the release process, we will automatically update the .po files using the GNU gettext family of tools (msgfmt, etc.)

**Review by Paul-Licameli — CHANGES_REQUESTED** [review]

File: src/menus/PluginMenus.cpp [hunk 2]
```diff
@@ -1215,7 +1215,7 @@ BaseItemSharedPtr GenerateMenu()
    ( FinderScope{ findCommandHandler },
    Menu( wxT("Generate"), XXO("&Generate"),
       Section( "Manage",
-         Command( wxT("ManageGenerators"), XXO("Add / Remove Plug-ins..."),
+         Command( wxT("ManageGenerators"), XXO("Plugin Manager"),
```
Comment (Paul-Licameli) [inline src/menus/PluginMenus.cpp:1234]: The first commit is simple but I think "..." is not supposed to be omitted from the new texts.  See the conversation at the Issue page.

File: src/menus/PluginMenus.cpp [hunk 3]
```diff
@@ -803,6 +813,12 @@ void OnManageEffects(const CommandContext &context)
    DoManagePluginsMenu(project);
 }
 
+void OnAddRealtimeEffects(const CommandContext& context)
+{
+   auto& project = context.project;
+   DoShowRealtimeEffectsSidePanel(project);
+}
```
Comment (Paul-Licameli) [inline src/menus/PluginMenus.cpp:820]: No objections to the third commit.

**Comment by Paul-Licameli** [comment]
That's all!

File: locale/af.po (hunk 1 above)
Comment (ksoze95) [reply locale/af.po]: Commit dropped.

File: src/menus/PluginMenus.cpp (hunk 2 above)
Comment (ksoze95) [reply src/menus/PluginMenus.cpp]: I have reintroduced the "..."

Comment (ksoze95) [reply src/menus/PluginMenus.cpp:1234]: I jumped the gun.  I have now removed "..." due to @LWinterberg comment on the issue page.

**Review by Paul-Licameli — APPROVED** [review]

**Pull request merged**